python3/raygun4py/utilities.py

```diff
@@ -30,6 +30,10 @@ def filter_keys(filtered_keys, object):
                     if sanitised_key in key:
                         iteration_target[key] = "<filtered>"
 
+    if isinstance(object, raygunmsgs.RaygunMessage):
+        object.__dict__.update(iteration_target)
+        return object
+
```

Comment:
The bug was that we returned the `object.__dict__` instead of the object when the instance was a `RaygunMessage`. So we need to check the original object type and if it was a `RaygunMessage`, then return it.